src/vs/workbench/api/browser/mainThreadComments.ts

```diff
@@ -212,8 +218,8 @@ export class MainThreadCommentController {
 		return this._features.options;
 	}
 
-	private readonly _threads: Map<number, MainThreadCommentThread> = new Map<number, MainThreadCommentThread>();
-	public activeCommentThread?: MainThreadCommentThread;
+	private readonly _threads: Map<number, MainThreadCommentThread<IRange | ICellRange>> = new Map<number, MainThreadCommentThread<IRange | ICellRange>>();
```

Comment:
Should `MainThreadCommentController` also be templated so that `IRange | ICellRange` can be a `T`?